internal/gps/error.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2017 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package gps
+
+import (
+	"bytes"
+	"fmt"
+)
+
+type errorSlice []error
```

Comment:
i'm a little uneasy about giving this its own file. i appreciate that it doesn't really fit where it was, but...well, we already have a bunch of antipatterns with errors, and i think it's becoming increasingly harmful.

no change to make here, just making a note of it.